src/pivot-table/components/PivotTable.tsx

```diff
@@ -123,6 +127,14 @@ export const StickyPivotTable = ({
   const headerCellRowHightCallback = useCallback(() => headerCellHeight, [headerCellHeight]);
   const contentCellRowHightCallback = useCallback(() => contentCellHeight, [contentCellHeight]);
 
+  useEffect(() => {
+    const element = verticalScrollableContainerRef.current;
+    if (element) {
+      // element.scrollLeft = scrollLeft;
+      element.scrollTop = scrollTopPartially;
```

Comment:
I definitely need to add a similar property for scrolLeft and other conditions if there are more, but maybe we can focus on vertical scroll for this PR